src/components/PasswordHashVerifier.js

```diff
@@ -16,6 +17,34 @@ const PasswordHashVerifier = () => {
       ...values,
       [name]: value,
     });
+    if(e.target.value === "bcrypt") {
```

Comment:
@Rachana21Reddy , writing conditions with only "if" vs writing with if-else is that even if the first condition matches, code must be evaluated with all "if"s . In case of if-else, once a match is found, rest is ignored. In this scenario, if-else performs better. Another Idea is to use a simple JSON object and with keys as algorithm and value as hash. Your lines of code improve and you can add more to JSON object in the future.